src/components/app/consts.ts

```diff
@@ -4,4 +4,12 @@ export enum GameState {
     Ended,
 }
 
-export const TURN_TIME = 10; // SETTING 10 FOR DEBUGGING PURPOSES
\ No newline at end of file
+export const TURN_TIME = 10; // SETTING 10 FOR DEBUGGING PURPOSES;
+
+const OpenerCategory = {
+    Funny: 'funny',
+    Mystery: 'mystery',
+    Random: 'random'
+} as const;
+
+export type OpenerCategory = (typeof OpenerCategory)[keyof typeof OpenerCategory];
```

Comment:
```suggestion
export type OpenerCategory = "random" | "mystery" | "funny;
```

Reply:
In my opinion we should move it to the Game interface , you can always do Game['openerCategory'] if you need a specific property type